x/interchainstaking/keeper/intent_test.go

```diff
@@ -334,7 +334,8 @@ func (suite *KeeperTestSuite) TestAggregateIntent() {
 				icsKeeper.SetDelegatorIntent(ctx, &zone, intent, false)
 			}
 
-			_ = icsKeeper.AggregateDelegatorIntents(ctx, &zone)
+			err := icsKeeper.AggregateDelegatorIntents(ctx, &zone)
+			suite.NoError(err)
```

Comment:
@joe-bowman after check for err the test failed here, is this a problem?
```
Received unexpected error:
ordinalized intent sum is zero, this may happen if no claims are recorded
```

Reply:
I think we can leave it for now; and will open a new issue to combine this and the commented out test below (TestAggregateIntentWithPRClaims) which should test aggregate intent with the PR claims set.